project/Build.scala

```diff
@@ -129,7 +135,7 @@ object Build extends sbt.Build {
                         ),
         packExclude := Seq(fsio.id, examples_kafka.id, sol.id, wordcount.id),
         packResourceDir := Map(baseDirectory.value / "conf" -> "conf"),
-        packExpandedClasspath := true,
+        packExpandedClasspath := false,
```

Comment:
The classpath should NOT be expanded.
Otherwise, the classpath maybe too long. On windows, it will report "command line too long."